test/integration/test_workflow_refactoring.py

```diff
@@ -15,13 +15,13 @@
 )
 from galaxy.workflow.refactor.schema import RefactorActionExecutionMessageTypeEnum
 from galaxy_test.base.populators import WorkflowPopulator
-from galaxy_test.base.uses_shed import UsesShed
 from galaxy_test.base.workflow_fixtures import (
     WORKFLOW_NESTED_RUNTIME_PARAMETER,
     WORKFLOW_NESTED_SIMPLE,
     WORKFLOW_NESTED_WITH_MULTIPLE_VERSIONS_TOOL,
 )
 from galaxy_test.driver import integration_util
+from galaxy_test.driver.uses_shed import UsesShed
```

Comment:
I think `UsesShedApi` should be enough for what's used below.

Reply:
I assumed these integration tests need the tool shed configs setup. How can ```self.install_repository("iuc", "compose_text_param", "feb3acba1e0a")``` without a tool sheds config file and such.

Reply:
Well it does work... but is it ... shouldn't be? Are those tools installing in default locations that would clash with running Galaxies or perhaps driver util is setting up more than I would assume by default but I feel like the contract should be that it doesn't.

Reply:
I guess this works now because we're replacing the whole mutable directory for these tests. 

shed_tool_config_file: /var/folders/1d/bs7mkcjx4t79x1rgbr2ns5br5ng4yk/T/tmpza_iy1ur/tmpnsuuc6o6/shed_tool_conf.xml ['/Users/jxc755/workspace/galaxy/test/functional/tools/samples_tool_conf.xml']

I will refactor it I suppose.